src/methods/gas_price.rs

```diff
@@ -1,3 +1,57 @@
+//! Returns the gas price for a specific block_height or block_hash.
+//!
+//! ## Examples
+//!
+//! Returns the gas fees for this block:
+//! <https://explorer.near.org/blocks/6atGq4TUTZerVHU9qWoYfzXNBg3K4C4cca15TE6KfuBr>
+//!
+//! - `BlockId::Height`
+//!
+//!     ```
+//!     # use near_jsonrpc_client::{methods, JsonRpcClient};
+//!     use near_primitives::types::BlockId;
+//!
+//!     # #[tokio::main]
+//!     # async fn main() -> Result<(), Box<dyn std::error::Error>> {
+//!     let client = JsonRpcClient::connect("https://archival-rpc.mainnet.near.org");
+//!
+//!     let request = methods::gas_price::RpcGasPriceRequest {
+//!         block_id: Some(BlockId::Height(48309771)),
```

Comment:
...isn't this height

Reply:
Yeah, this is height, the example for `hash` is below it

Reply:
Do you want the `hash` example to come first?

Reply:
Both comments go together.

> This hash: `https://explorer.near.org/blocks/6atGq4TUTZerVHU9qWoYfzXNBg3K4C4cca15TE6KfuBr`
> Isn't this height: `48309771`

Reply:
The height should be `61512623`.

Reply:
True, my bad